druntime/src/core/thread/types.d

```diff
@@ -41,8 +41,9 @@ version (GNU)
 }
 else
 {
-    // this should be true for most architectures
-    enum isStackGrowingDown = true;
+    version (X86) enum isStackGrowingDown = true;
+    else version (X86_64) enum isStackGrowingDown = true;
+    else static assert(0, "Dunno how the stack grows on this architecture.");
```

Comment:
I think we should stick to the appropriate tone and language when it comes to error messages.